fili-core/src/test/groovy/com/yahoo/bard/webservice/table/availability/PermissiveAvailabilitySpec.groovy

```diff
@@ -49,15 +50,14 @@ class PermissiveAvailabilitySpec extends Specification  {
 
```

Comment:
This test is incorrect to design.  Create a 'long_interval' column and confirm that even when not selecte that the getAvailableIntervals()l is always the long_interval or the union of the long_interval with any non overlapping intervals.